bindings/java/examples/basic-app/src/main/java/org/example/ExampleApp.java

```diff
@@ -46,7 +46,7 @@ public ExampleApp() {
         // String mnemonic = manager.generateMnemonic();
         
         // null means "generate one for me"
-        manager.storeMnemonic(AccountSignerType.STRONGHOLD, null);
+        manager.storeMnemonic(Accountsecret_manager.STRONGHOLD, null);
```

Comment:
What kind of case is that ? It's horrible :trollface:

Reply:
That's don'tCare_because binding will be created new anyways and all this will be gone :trollface:

Reply:
That's don'tCare_because binding will be created new anyways and all this will be gone :trollface: